content/articles/introduction-to-kernel-methods-in-machine-learning/Index.md

```diff
@@ -0,0 +1,169 @@
+# Introduction to Kernel Methods in Machine Learning
+
+### Prerequisites
+- The Reader should have basic knowledge in machine learning [Click here](https://www.digitalocean.com/community/tutorials/an-introduction-to-machine-learning) to read more about machine learning basics.
+- The reader should be able to understand basic terms like Support Vector Machine, classifications, unsupervised learning and supervised learning.

```

Comment:
reword to "- The reader should understand basic terms like Support Vector Machine, classifications, unsupervised learning, and supervised learning."